js/embed.js

```diff
@@ -9,6 +9,14 @@ import { renderMime } from './renderMime'
 import 'font-awesome/css/font-awesome.css'
 import './widgets.css'
 
+// TODO - put this into singleton class to avoid
+// standalone global variable
+let kernelGlobal = null;
+
+export async function killProcess(baseUrl, wsUrl) {
+    let result = await kernelGlobal.shutdown()
```

Comment:
TODO for me: wrap in try/catch block, to audit success/failure of this shutdown

Reply:
FYI -- Putting `console.log(result)` does work, AFAICT. It just logs undefined because `Kernel.shutdown()` returns `Promise<void>`, which, once awaited, might log undefined?

Reply:
ah, I see- I'm thinking since it returns Promise<void>, it will be cleaner then to remove the 'result' variable here? I'll retest to make sure this logic still works with that change